### PR TITLE
Fix issue 264

### PR DESCRIPTION
**Pull request by leonmatt**
This PR simplifies the code for the VmallocStater class.

It targets issue #264.

### CODE REVIEW

File: stats/kernel-selftests [hunk 1]
```diff
@@ -1359,12 +1359,13 @@ class VmallocStater < Stater
     # [  223.187358] Summary: kvfree_rcu_2_arg_vmalloc_test passed: 1 failed: 0 repeat: 1 loops: 1000000 avg: 6333619 usec
     # [  223.198414] All test took worker0=441656368064 cycles
     when /Summary: (.+) passed: (.+) failed: (.+) repeat: (.+) loops: (.+) avg: (.+) usec/
-      if $3.to_i.positive?
-        @tmp_stats[$1] = 'fail'
-      elsif $2.to_i.positive?
+      result = 'fail'
+      if $3.to_i == 0
```
Comment (rli9) [inline stats/kernel-selftests:1363]: thanks, here can use $3.to_i.zero?

Comment (leonmatt) [reply stats/kernel-selftests:1363]: Yep, I will get that finished.

**Comment by rli9** [comment]
thanks, could you help merge the commits to 1 single one, and do a force push to the branch?

**Comment by leonmatt** [comment]
> thanks, could you help merge the commits to 1 single one, and do a force push to the branch?

Done

**Comment by rli9** [comment]
thanks for the patch

**Pull request merged**